components/Form.tsx

```diff
@@ -0,0 +1,40 @@
+
+const Form =()=>{
+    return(
+<div className="max-w-md mx-auto p-4">
+  <form>
+    <div className="mb-4">
+      {/* <label className="block text-gray-700 text-sm font-bold mb-2" htmlFor="caseName">
+        Case Name
+      </label> */}
```

Comment:
Remove comments if you not need them

Reply:
Just a bad habit, leaving it commented until i decide what i'll do with it. Basically was just looking design wise is it better with or without. But I'll leave the habit in the past because it's not a good practice. Thanks for pointing it out though, if we are gonna do things might as well do them the right :).